### ISSUE TITLE
skills-int-client-vue ProxiedUser component blank when run outside of e2e test

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-client#108]

**Comment by sudo-may** [comment]
PR: https://github.com/NationalSecurityAgency/skills-client/pull/109

**Comment by rmmayo** [comment]
@mosefer  - having trouble testing this...looks like there are two issue (one definitely is a separate unrelated ticket):

1. Doesn't seem like the integration server is being deployed with the latest docker image.  I went to test on integration, it was broken there.  I looked for the code changes in `main.js` and didn't see any of the recent code changes.  I then went to the integration server and inspected the docker image - it was using the `latest` tag the deployed image, but the build date and time stamp were from 3 months ago ().
2. I then went to the test EC2 machine and redeployed the image built yesterday there and I get an error ().  It looks like there's no wat for me to override the `skills.service.proxiedProxyId` value using env vars in the docker-compose.yml because it's hard-coded in the `start.sh` script.
